web/package.json

```diff
@@ -87,6 +87,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-i18next": "11.18.6",
+    "react-markdown": "^8.0.4",
```

Comment:
```suggestion
    "react-markdown": "8.0.4",
```

Reply:
Please do this locally (or pull in from github after committing suggestion) and then do yarn again to regenerate your yarn.lock too

Reply:
Did you do yarn again locally? I was expecting your yarn.lock to change to reflect setting the exact version, but doesn't seem like it...
If you did it, but nothing changed please ignore this and resolve this comment